.gitignore

```diff
@@ -88,3 +88,4 @@ core
 config-ver.h
 test-driver
 version
+__pycache__
```

Comment:
Shouldn't it be <test/__pycache__> instead?

Also, do we ignore builddir/ and doc/?

I see src/.dirstamp and src/x11/.dirstamp in the untracked files list too. Sorry for the noise :)

Reply:
gitignore by default matches patterns anywhere, so `__pycache__` will match any such directory anywhere. Since there's no good reason to ever commit `__pycache__`, we might as well not limit ourselves to a single directory - even though we only ever expect that have that file.

`.dirstamp` is ... I think from Make? We should probably add that too then, I'll let you file a separate PR for that though :)

Reply:
I only see the .dirstamp files under the repo I cloned a while back. I cloned a new one. I don't see the files any more. I don't know why. I think we don't need to worry about them now. So, feel free to merge your patch.